backend/globaleaks/handlers/admin/https.py

```diff
@@ -124,6 +135,7 @@ def db_serialize(store):
         prv_fact = PrivateFactory(store)
 
         return {
+            #TODO(remove) places key material in memory
```

Comment:
instead #TODO lets try to see if we have a ticket tracking this, improve or create the ticket with notes on the topic.